tox.ini

```diff
@@ -74,9 +74,15 @@ commands =
 deps = -r{toxinidir}/docs/requirements.txt
 commands = python3 setup.py build_sphinx -W
 
+[testenv:vulture]
+deps =
+    vulture
+commands =
+    vulture --min-confidence=100 --exclude test/configs/syntaxerr.py libqtile test
```

Comment:
So this will fail tests only when it finds something it is 100% sure about?

Reply:
Yes, exactly. Unfortunately due to the nature of python, that is not very much stuff.